apps/site/assets/ts/components/Headsign.tsx

```diff
@@ -45,22 +45,29 @@ const renderTrainName = (trainName: string): ReactElement<HTMLElement> => (
   <div className="m-tnm-sidebar__headsign-train">{trainName}</div>
 );
 
-const crTime = (data: PredictedOrScheduledTime): ReactElement<HTMLElement> =>
-  timeForCommuterRail(data, "m-tnm-sidebar__time-number");
-
 const renderTimeCommuterRail = (
   data: PredictedOrScheduledTime,
   modifier: string
-): ReactElement<HTMLElement> => (
-  <div
-    className={`m-tnm-sidebar__time m-tnm-sidebar__time--commuter-rail ${modifier}`}
-  >
-    {crTime(data)}
-    <div className="m-tnm-sidebar__status">
-      {`${statusForCommuterRail(data) || ""}${trackForCommuterRail(data)}`}
+): ReactElement<HTMLElement> => {
+  const status = statusForCommuterRail(data) || "";
+  return (
+    <div
+      className={`m-tnm-sidebar__time m-tnm-sidebar__time--commuter-rail ${modifier} ${
+        status === "Scheduled" ? "text-muted" : ""
+      }`}
+    >
+      {timeForCommuterRail(
+        data,
+        `m-tnm-sidebar__time-number ${
+          status === "Canceled" ? "strikethrough" : ""
```

Comment:
I think a quick test for this visual behavior could be useful to prevent regressions.